packages/features/embed/Embed.tsx

```diff
@@ -385,7 +390,10 @@ const EmailEmbedPreview = ({
                 Object.keys(selectedDateAndTime)
                   .sort()
                   .map((key) => {
-                    const selectedDate = dayjs(key).tz(timezone).format("dddd, MMMM D, YYYY");
```

Comment:
Key is date of the selectedDateAndTime so when converting to timezone, it wasn't considering time of the slot, only the date. This was causing slots to go to previous date

Eg. if selected slot is '2024-07-25T01:51:59.881Z', key would be 2024-07-25.

`const selectedDate = dayjs('2024-07-25').tz("Asia/Kolkata").format("dddd, MMMM D, YYYY");` 
would be Saturday, July 27, 2024

But the same thing on changing to Asia/Karachi would show previous date
`const selectedDate = dayjs('2024-07-25').tz("Asia/Karachi").format("dddd, MMMM D, YYYY");` 
would be Friday, July 26, 2024